tendrl/ceph_integration/manager/cluster_monitor.py

```diff
@@ -366,6 +367,122 @@ def on_sync_object(self, data):
                 " received for %s" % data['type']
             )
 
+    def _get_utilization_data(self):
+        from ceph_argparse import json_command
```

Comment:
@r0h4n ceph.py returns data formatted with json_loads_byteified(). In my case I need raw data to process further. I remember now that I checked ceph.py but decided to use json_command directly in this specific case.